tools/clang/tools/dxcompiler/DXCompiler.cpp

```diff
@@ -106,8 +106,8 @@ BOOL WINAPI DllMain(HINSTANCE hinstDLL, DWORD Reason, LPVOID reserved) {
     DxcEtw_DXCompilerShutdown_Start();
     DxcSetThreadMallocToDefault();
     ::hlsl::options::cleanupHlslOptTable();
-    ::llvm::sys::fs::CleanupPerThreadFileSystem();
     ::llvm::llvm_shutdown();
+    ::llvm::sys::fs::CleanupPerThreadFileSystem();
```

Comment:
This may fix the issue, but I'm still concerned that we are attempting to create a file in a destructor during `llvm_shutdown()` to write timer results.  Especially from the perspective of the dxcompiler.dll.